pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -35,14 +35,14 @@
                                        time_code, setup_results_object, process_objective, lower_logger_level_to)
 from pyomo.contrib.mindtpy.initialization import MindtPy_initialize_main
 from pyomo.contrib.mindtpy.iterate import MindtPy_iteration_loop
-from pyomo.contrib.mindtpy.util import model_is_valid, setup_solve_data
+from pyomo.contrib.mindtpy.util import model_is_valid, set_up_solve_data
 from pyomo.core import (Block, ConstraintList, NonNegativeReals,
                         Set, Suffix, Var, VarList, TransformationFactory, Objective, RangeSet)
 from pyomo.opt import SolverFactory
 from pyomo.contrib.mindtpy.config_options import _get_MindtPy_config, check_config
 
 logger = logging.getLogger('pyomo.contrib.mindtpy')
-
+logging.basicConfig(format='%(message)s')
```

Comment:
This is almost certainly not a good idea: importing Mindtpy now reconfigures all logging across Python.

Reply:
Actually, I only want to change the logging format in MindtPy. Do you have suggestions for this?

Reply:
Btw, do we need `logger = logging.getLogger('pyomo.contrib.mindtpy')` in each file of MindtPy? 
I remember GDPOpt previously had similar codes. But they are removed now. 
I am not very familiar with the `logging` package. Any comments or suggestions are welcomed. Thanks.